readthedocs/builds/forms.py

```diff
@@ -131,6 +135,19 @@ def __init__(self, *args, **kwargs):
             (TAG, TAG_TEXT),
         ]
 
+        # Only list supported actions
```

Comment:
```suggestion
        # Remove privacy actions not available in community
```

Adds a little more context about what "supported" means.